src/mmw/sass/base/_map.scss

```diff
@@ -157,6 +160,10 @@
             background-color: #ccc;
         }
     }
+    .leaflet-touch & {
```

Comment:
Thanks to @jfrankl for teaching me this neat trick! This allows accounting for `.leaflet-touch` coming from the outside scope, since that class is applied to the `.map-container` tag.